jenkins_pipelines/scripts/test_environment_cleaner/delete_s390_clients.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+##### Manually delete the s390 using delete_s390_guest command #####
+# Check if the main.tf file is provided as an argument
+if [ "$#" -ne 1 ]; then
+  echo "Usage: $0 <path_to_main.tf>"
+  exit 1
+fi
+
+# Get the main.tf file from the argument
+MAIN_TF_FILE=$1
+
+# Check if the file exists
+if [ ! -f "$MAIN_TF_FILE" ]; then
+  echo "Error: File $MAIN_TF_FILE does not exist."
+  exit 1
+fi
+
+# Extract user IDs from the main.tf file
+USERIDS=$(grep -A 1 "userid" "$MAIN_TF_FILE" | awk -F'"' '/userid/ {print $2}')
+echo "Extracted userid: $USERIDS"
+# Iterate through each userid and delete the s390 clients
+for USERID in $USERIDS; do
+  echo "Deleting client with userid: $USERID"
+  /usr/local/bin/delete_s390_guest "$USERID"
```

Comment:
That binary is already part of our IaC in the infrastructure repo?

Reply:
No, but I can add it.

Reply:
Thank you Eric

Reply:
After the workaround is removed (hopefully before it snows in hell...) it won't be a dependency anymore for our pipelines.